site/_includes/holder.html

```diff
@@ -0,0 +1,41 @@
+{%- comment -%}
+Usage: {% include holder.html size="64x64" %},
+size - null (default)
+optional: text - "[width]x[height]" (default)
+optional: bg - "#777" (default)
+optional: fg - "#ddd" (default)
+optional: text_size - "13" (default)
+{%- endcomment -%}
+
+{%- assign size = include.size | split: "x" -%}
+{%- assign width = size[0] | default: "800" -%}
+{%- assign height = size[1] | default: "400" -%}
+
+{%- capture text -%}
+  {%- if include.text == null -%}
+    {{ width }}x{{ height }}
+  {%- else -%}
+    {{ include.text }}
+  {%- endif -%}
+{%- endcapture -%}
+
+{%- assign background = include.bg | default: "#777" -%}
+{%- assign text_color = include.fg | default: "#ddd" -%}
+{%- assign text_size = include.text_size | default: "13" -%}
+
+{%- capture svg -%}
+<svg xmlns='http://www.w3.org/2000/svg' width='{{ width }}px' height='{{ height }}px' viewBox='0 0 {{ width }} {{ height }}'>
+  <path fill='{{ background }}' d='M0 0h{{ width }}v{{ height }}H0z'/>
+  <text fill='{{ text_color }}' font-family='sans-serif' font-size='{{ text_size }}' x='50%' y='50%' alignment-baseline='middle' text-anchor='middle'>{{ text }}</text>
+</svg>
+{%- endcapture -%}
+{%- assign svg_encoded = svg
```

Comment:
Thx, for update the naming. Naming can be always improved. I've intentionally didn't escape spaces. See the findings of [this](https://css-tricks.com/probably-dont-base64-svg/) and [this](https://codepen.io/tigt/post/optimizing-svgs-in-data-uris) to keep the characters as low as possible.

Liquid has some filters, like [`escape`](
https://shopify.github.io/liquid/filters/escape/), [`url_encode`](https://shopify.github.io/liquid/filters/url_encode/) but they don't fit perfectly because I wanted to just replace the necessary chars to keep the svg_normalize string at a minimum.

Reply:
The space escape is fixed with my last patch.